docs/ref-migrate-0.13-0.20.md

```diff
@@ -176,13 +176,13 @@ Consider the Ergo logic for the [acceptance of delivery](https://templates.accor
       throw ErgoErrorResponse{ message : "Transaction time is before the deliverable date." }
     ;
 
-    let dur = 
+    let dur =
       Duration{
         amount: contract.businessDays,
         unit: "days"
       };
     let status =
-      if isAfter(now(), addDuration(received, dur))
+      if isAfter(now, addDuration(received, dur))
```

Comment:
No! 😃 do not change it, that's the old use of `now` which is fixed in the version below!!

Reply:
@jeromesimeon so in the previous version, there was a 'now' and a 'now()', and in `0.20` there is only 'now()'?

Reply:
That's right, @4katie.
To be precise `now` is just a variable. It's still there but can only be used if you do `let now = someExpression; ...`, just like any other variable. `now()` was always the right way to access current time, so people won't be able to rely on that bug.

Since this is the migration guide, that's one of the examples that we should document.